.coveragerc

```diff
@@ -0,0 +1,2 @@
+[report]
+omit = */test_*.py
```

Comment:
probably add `mediacloud/mediawords/test/*` to this

Reply:
In some cases tests are being tested too ;), e.g. we test the `HashServer` class that we use for mock HTTP servers.

Reply:
cool! just saw it included on coveralls with not much coverage and thought it might be a mistake